src/main/java/net/coderbot/patchwork/manifest/forge/AccessTransformerList.java

```diff
@@ -0,0 +1,73 @@
+package net.coderbot.patchwork.manifest.forge;
+
+import net.coderbot.patchwork.mapping.TsrgMappings;
+
+import java.io.ByteArrayInputStream;
+import java.lang.reflect.Field;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import net.fabricmc.mappings.Mappings;
+import net.fabricmc.mappings.MappingsProvider;
+import net.fabricmc.tinyremapper.IMappingProvider;
+import net.fabricmc.tinyremapper.TinyRemapper;
+import org.objectweb.asm.commons.Remapper;
+
+public class AccessTransformerList {
+
+	private List<AccessTransformerEntry> entries;
+	public AccessTransformerList(List<AccessTransformerEntry> entries) {
+		this.entries = entries;
+	}
+
+	public List<AccessTransformerEntry> getEntries() {
+		return entries;
+	}
+
+	// TODO fails mysteriously when unable to remap
+	public static AccessTransformerList parse(Path accessTransformer,
```

Comment:
I think this should instead simply read the AT file into the list of entries. Then, make a separate `remap(Remapper remapper)` function that people can use to remap the list of entries.

This would allow reuse of the Remapper from TinyRemapper in the future, since creating a new TinyRemapper is a very expensive operation performance wise.

Reply:
Leaving this review open because I haven't made them reused yet.